### ISSUE TITLE
Easter Egg Surprise

### ISSUE THREAD

**Issue by wuworkshop** [codebuddiesdotorg/cb-v2-scratch#85]
Might be fun to add an easter egg to the app, just as a nice surprise to users. Like maybe when a user types the Konami code they see something cool or funny.

Might I suggest 17 on http://tympanus.net/Development/Animocons/ ? :smile: 

Example of Konami code implementation (please note it's a VERY old post) - http://www.paulirish.com/2009/cornify-easter-egg-with-jquery/#comment-68011

**Comment by sharynneazhar** [comment]
This sounds pretty awesome 😄. Are we still interested in implementing this?

**Comment by adachiu** [comment]
@sharynneazhar yep we sure are! We are most likely implementing a unicorn. Actually, let me label this as in progress.

**Comment by mahamshahid18** [comment]
Hello. I'd like to work on this, sounds fun 😄 @wuworkshop

**Comment by billglover** [comment]
@mahamshahid18 are you still working on this issue? If so, do you need any help/support?

If not, that's ok. Let me know and I'll remove the 'claimed' tag.

**Comment by billglover** [comment]
I've moved this back to the backlog and removed the claimed label.
